README.md

```diff
@@ -1,20 +1,157 @@
+
+1) Raising awareness of inherent unfairness in model predictions among those involved in developing AI applications by demonstrating harms visited on vulnerable groups
+
+2) Providing state-of-the-art techniques to mitigate the observed unfairness
+
+
+At the assessment phase, Fairlearn provides a dashboard with a rich set of visualizations that uses a set of disparity and accuracy metrics and user-defined protected attributes (e.g., age, gender, ethnicity) to monitor the accuracy and fairness of a model’s predictions.
+
+
+At the mitigation phase, it incorporates state-of-the-art unfairness mitigation techniques to mitigate the unfairness observed at the previous assessment phase. Fairlearn’s mitigation stage supports regression and binary classification models trained on tabular data.
+
+
+
+# <a name="target"></a>
+# Target Audience
+
+
+
+1. Machine Learning Fairness Researchers: ML researchers can easily add their unfairness mitigation techniques to this repository and use the assessment toolkit to load their original and refined model predictions to get insights into how much they have improved the disparity metrics of interest.
+
+
+2. Developers/ ML Engineers/ Data Scientists: Having a variety of curated unfairness mitigations techniques in one place makes it easier for data scientists and (ML) developers to experiment with different unfairness techniques in a seamless manner. The set of rich interactive visualizations allow developers and data scientists to easily visualize the unfairness happening in their machine learning models instead of wasting time and effort on generating customized visualizations.
+
+3. Business Executives: The set of provided visualizations are beneficial for raising awareness among those involved in developing AI applications, allow them to audit model predictions for potential unfairness, and establish a strong governance framework around the use of AI applications.
+
+# <a name="assessment"></a>
+# Fairness Assessment Dashboard 
+The assessment dashboard automatically analyzes a model’s predictions, provides user with a set of insights into how the model is treating different buckets (e.g., female, male, other gender) of a protected attribute/feature (e.g., gender).  
+
+
+
+These insights are calculated via a set of disparity metrics that showcase:
+
+1) [Disparity in Accuracy]: How the model accuracy differs across different buckets of a selected protected attribute (e.g., how accuracy of the model differs for "females" vs. "males" vs. "other gender" data points),
+
+2_ [Disparity in Predictions] How the model predictions differ across different buckets of a selected protected attribute  (e.g., how many "females" have received prediction `Approved` on their loan application in contrast to "males" and "other gender" data points?). 
```

Comment:
Should that be "2)" ?

Again, can this be related to a well publicised failure. Although it would be safer to use one from MS (if we have one) rather than the Amazon hiring one.

Also, is this "Disparity in Opportunity?"

Reply:
In markdown it's better to use
1.
1.
1.

and it'll handle the numbering automagically ;-)

Reply:
I don't see a hiring scenario here, what are you referring to?